lib/workers/repository/onboarding/pr/__snapshots__/errors-warnings.spec.ts.snap

```diff
@@ -4,7 +4,7 @@ exports[`workers/repository/onboarding/pr/errors-warnings getDepWarnings() retur
 "
 ---
 
-### ⚠️ Dependency Lookup Warnings ⚠️
+### ⚠ Dependency Lookup Warnings ⚠
```

Comment:
🤔  Don't like these icons 😕

Reply:
⚠️ These are rendered yellow in web pages, while looking confusing in the editor (you can try copy-paste into comment and switch between write/preview)

Reply:
⚠

Reply:
ok, 🤦‍♂️